lib/view_model/send/send_view_model.dart

```diff
@@ -127,15 +130,17 @@ abstract class SendViewModelBase with Store {
   bool get isReadyForSend => _wallet.syncStatus is SyncedSyncStatus;
 
   @computed
-  ObservableList<Template> get templates => sendTemplateViewModel.templates;
+  List<Template> get templates => sendTemplateViewModel.templates
+      .where((template) => template.cryptoCurrency == _wallet.currency.title)
```

Comment:
For compare strings like that better to normalize them, for example be sure that they are in needed register. Please force them to lower case when comparing string like those.

Reply:
Alright